app/views/layouts/application.slim

```diff
@@ -33,3 +33,9 @@ html lang="en"
     = javascript_include_tag :application
     - if Rails.env.production?
       = javascript_include_tag "//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-55ad6e555f728b0f", async: true
+    - unless Rails.env.test?
```

Comment:
Intercom has a sandbox mode (which provides you with a different API key) which you can use in other environments to test whatever you want. But we don't really need that here, at least right now

Using intercom with the production API key should be constricted to the production environment, otherwise you'll end up publishing development data, which is definitely something we don't want